src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -467,7 +467,7 @@ define(function (require, exports, module) {
 
                 var dragSource = collection.pluck(this.state.dragTargets, "id");
 
-                flux.actions.layers.reorder(doc, dragSource, dropIndex)
+                return flux.actions.layers.reorder(doc, dragSource, dropIndex)
```

Comment:
I know you didn't add this `then` handler, but it's a little odd that it's in the JSX instead of as part of some other action. I only mention it because there is not actually any reason why this method needs to return a promise.